setup.py

```diff
@@ -42,7 +43,7 @@ def readme():
     packages=find_packages(),
     package_data={"protopipe": ["aux/example_config_files/protopipe/analysis.yaml"]},
     include_package_data=True,
-    install_requires=["ctapipe"],
+    install_requires=["ctapipe==0.9.1", "jupyterlab", "pyirf", "vitables"],
```

Comment:
is jupyterlab really required? or only to generate docs?

Reply:
I think that is the incorrect way to do it. Pip is not meant to making environments, only for installing required dependencies. For environments you need to use `pipenv` or `poetry`, or just provide a `requirements.txt` file that is the output of `pip freeze`.    In the package itself (in setup.py) you should only list the bare minimum to allow it to work.  Otherwise you will run into dependency resolution problems later. 

So here you should have only ctapipe and pyirf as dependencies.